### PR TITLE
[C-927] Move sign up to native

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Move sign up to react native.

Delete lots of code.

Requires sdk change: https://github.com/AudiusProject/audius-protocol/pull/3788

Caveats:
- Oauth not working yet
- Solana user bank creation console errors during sign up, but not blocking. will address separately

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Locally with sim -- created staging account!

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-927

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-927

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-927

File: packages/web/src/common/store/pages/signon/sagas.js [hunk 1]
```diff
@@ -566,6 +568,7 @@ function* followCollections(collectionIds, favoriteSource) {
 }
 
 function* followArtists() {
+  console.log('follow artists')
```
Comment (raymondjacobson) [inline packages/web/src/common/store/pages/signon/sagas.js:571]: remove

File: packages/web/src/common/store/pages/signon/sagas.js [hunk 2]
```diff
@@ -585,6 +588,7 @@ function* followArtists() {
     ]
     for (const userId of userIdsToFollow) {
       yield put(socialActions.followUser(userId))
+      console.log('put actions', userId)
```
Comment (raymondjacobson) [inline packages/web/src/common/store/pages/signon/sagas.js:591]: remove

File: packages/mobile/src/screens/signon/ProfileManual.tsx [hunk 3]
```diff
@@ -561,19 +422,16 @@ const ProfileManual = ({ navigation, route }: ProfileManualProps) => {
...
-                        if (handleStatus === 'editing') {
-                          setHandleDebounce(true)
-                        }
+                      clearTimeout(updateHandleConfirmationTimeout)
+                      updateHandleConfirmationTimeout = setTimeout(() => {
+                        // Show a spinner after a delay while the handle validates
+                        setShowHandleConfirmingSpinner(true)
                       }, HANDLE_VALIDATION_IN_PROGRESS_DELAY_MS)
-                      dispatch(signonActions.setHandleStatus('editing'))
+                      // dispatch(signonActions.setHandleStatus('editing'))
```
Comment (sliptype) [inline packages/mobile/src/screens/signon/ProfileManual.tsx:432]: Not needed?

File: packages/mobile/src/store/lifecycle/reducer.ts [hunk 4]
```diff
@@ -1,12 +1,6 @@
...
 
 import type { LifecycleActions } from './actions'
-import {
-  SIGNED_IN,
-  SIGNED_OUT,
-  ON_SIGN_UP,
-  FETCH_ACCOUNT_FAILED
-} from './actions'
 
 export type LifecycleState = {
```
Comment (sliptype) [inline packages/mobile/src/store/lifecycle/reducer.ts:5]: Can we just get rid of this whole reducer?

File: packages/web/src/pages/sign-on/SignOnProvider.tsx [hunk 5]
```diff
@@ -263,8 +263,6 @@ export class SignOnProvider extends Component<SignOnProps, SignOnState> {
   }
 
   finishSignup = () => {
-    const { sendWelcomeEmail, fields } = this.props
-    sendWelcomeEmail(fields.name.value)
```
Comment (sliptype) [inline packages/web/src/pages/sign-on/SignOnProvider.tsx:267]: Is this done elsewhere now?

**Review by sliptype — APPROVED** [review]
omg so much cleaner

File: packages/web/src/pages/sign-on/SignOnProvider.tsx (hunk 5 above)
Comment (raymondjacobson) [reply packages/web/src/pages/sign-on/SignOnProvider.tsx:267]: yep in the saga

File: packages/mobile/src/store/lifecycle/reducer.ts (hunk 4 above)
Comment (raymondjacobson) [reply packages/mobile/src/store/lifecycle/reducer.ts:5]: Wasn't sure if you needed it still. will delete!

Comment (sliptype) [reply packages/mobile/src/store/lifecycle/reducer.ts:5]: Just need the actions for use in sagas!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-927

File: packages/mobile/src/screens/signon/ProfileManual.tsx (hunk 3 above)
Comment (raymondjacobson) [reply packages/mobile/src/screens/signon/ProfileManual.tsx:432]: nope! thx

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-927

**Pull request merged**